Project/Modules/Blogs/Services/blogService.cs

```diff
@@ -0,0 +1,43 @@
+using blog_net_core.Project.Modules.Blogs.Model.Entities;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
```

Comment:
Document all the file you can use the <inheritdoc > Implemented using the blog service interface.</inheritdoc> at the method levels like 
![image](https://user-images.githubusercontent.com/63543622/146750648-ba83feb1-bf2f-45fa-a9f6-7fea30f42241.png)

Reply:
Resolved.